src/main.rs

```diff
@@ -30,6 +30,7 @@ compile_error!(
     "exactly one of the features \"native-roots\", \"webpki-roots\" or \"native-tls\" must be enabled"
 );
 
+use std::iter::once;
 use std::{env, process};
```

Comment:
Nit: Can you merge these imports for consistency?

```
use std::{env, iter::once, process};
```

Reply:
Merged.